src/docs/monarch-roles/mondo_technical_lead.md

```diff
@@ -0,0 +1,27 @@
+## Monarch role: Mondo technical lead
+
+Main responsibilities:
+
+1. Coordinate the development of the Mondo ingest pipeline
+1. Ensuring the ongoing Mondo ingest pipeline with external sources runs smoothly (devops)
```

Comment:
Dear Nico, if I may, since we are updating this documentation, I would like to kindly ask you to please update this list to make it all infinitive tense so each item describes the action to be taken. That is use 'ensure' instead of 'ensuring'. Thank you!

Reply:
```suggestion
1. Ensure the ongoing Mondo ingest pipeline with external sources runs smoothly (devops)
```

Reply:
I agree, great feedback. I have also made an issue about it for _all_ kinds of documentation #50